react-app/src/components/dialog/DeleteDialog.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+import { Box, TextField, Typography } from '@mui/material';
+import ConfirmDialog from './ConfirmDialog';
+
+interface IDeleteDialog {
+  open: boolean;
+  title: string;
+  message: string;
+  deleteText?: string;
+  onDelete: () => void;
+  onClose: () => void;
+}
+
+const DeleteDialog = (props: IDeleteDialog) => {
+  const { open, title, message, deleteText, onDelete, onClose } = props;
+  const [textFieldValue, setTextFieldValue] = useState('');
+  return (
+    <ConfirmDialog
+      open={open}
+      title={title}
+      onConfirm={onDelete}
+      onCancel={onClose}
+      confirmButtonText={deleteText}
```

Comment:
Should this have a default? Could be undefined.
```suggestion
      confirmButtonText={deleteText ?? 'Delete'}
```